client/src/componets/LikeDislikeButton/LikeDislikeButton.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { BsFillHandThumbsUpFill } from "react-icons/bs";
+import { BsFillHandThumbsDownFill } from "react-icons/bs";
+import "./LikeDislikeButton.css";
+
+function LikeDislikeButton({ rating, handleIncrement, handleDecrement }) {
```

Comment:
You've got one component here `LikeDislikeButton` that renders both the like and the dislike button. How can you break up this component into smaller parts that do one thing.

Reply:
Thank , i will try to put them in separate components so that it will be easier to undertand and access it